.eslintignore

```diff
@@ -67,18 +67,21 @@ packages/data-context/test/unit/codegen/files
 # community templates we test against, no need to lint
 system-tests/projects/create-react-app-configured/**/*
 system-tests/projects/create-react-app-unconfigured/**/*
+system-tests/projects/create-react-app-custom-index-html
```

Comment:
Not needed now but we could consider grouping all these community template projects w/ a common prefix like `template-` so it's easier to 1) ignore them for linting and 2) group them all together.

Reply:
Can we just lint ignore all system-test projects?

Reply:
Not sure, maybe, but a discussion and task for another PR, this is okay for now.